cdk/visit/lambda_code/register_user/register_user.py

```diff
@@ -0,0 +1,83 @@
+import json
+import boto3
+from boto3.dynamodb.conditions import Key
+import os
+import datetime
+
+
+class RegisterUserFunction():
+    """
+    This class wraps the function of the lambda so we can more easily test
+    it with moto. In production, we will continue to pass the stood-up
+    dynamodb table to the handler itself. However, when initializing this class,
+    we can choose to instead initialize it with a mocked version of the
+    dynamodb table.
+    """
+
+    def __init__(self, table):
+
+        if table is None:
+            # Default Behavior in Prod
+            # Get the service resource.
+            dynamodb = boto3.resource('dynamodb')
+            # Get the table name.
+            TABLE_NAME = os.environ["TABLE_NAME"]
+            # Get table objects
+            self.users = dynamodb.Table(TABLE_NAME)
+        else:
+            self.users = table
+
+    def addUserInfo(self, user_info):
+        # Get the current date at which the user registers.
+        timestamp = datetime.datetime.now()
+
+        response = self.users.put_item(
+            Item={
+                'PK': user_info['username'],
+                'SK': str(timestamp),
+                'firstName': user_info['firstName'],
+                'lastName': user_info['lastName'],
+                'Gender': user_info['Gender'],
+                'DOB': user_info['DOB'],
+                'Grad_date': user_info['Grad_Date'],
+                'Major': user_info['Major'],
+                'Minor': user_info['Minor']
```

Comment:
Nitpick but I would standardize on capitalization here. Looks like we have a mix of camelCase, PascalCase and Snake_Case. Becomes tricky to deserialize json with mixed casing because most languages have built in marshalers that expect uniform casing in objects.